pkg/kritis/review/review_test.go

```diff
@@ -91,6 +105,18 @@ func TestReviewGAP(t *testing.T) {
 			AttestationAuthorityNames: []string{"test", "test2"},
 		},
 	}}
+	// One policy without attestor:
```

Comment:
I think L99 above (comment explaining gapWithTwoAAs) is now incorrect.  There is an image which satisfies both auths.